### PR TITLE
 MOOR PACKAGES!

### PR DESCRIPTION
**Pull request by Commandcracker**
**com.github.release required**

### CODE REVIEW

File: basalt.lua [hunk 1]
```diff
@@ -1,19 +1,16 @@
...
 package.desc = "UI Framework for CC:Tweaked"
-package.version = "1.6.3"
+package.version = "1.6.4"
 package.instdat = {}
 package.instdat.repo_owner = "Pyroxenium"
 package.instdat.repo_name = "Basalt"
-package.instdat.repo_ref = "v1.6.3"
+package.instdat.repo_ref = "v1.6.4"
 package.instdat.filemaps = {}
 package.instdat.filemaps["basalt.lua"] = "/lib/basalt.lua"
```
Comment (tomodachi94) [inline basalt.lua:15]: Is there a minified Basalt available? It is pretty big.

File: brainfuck.lua [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+local package = {}
+package.name = "graph"
+package.desc = "Library for graphics functions."
+package.instdat = {}
+package.instdat.repo_owner = "9551-Dev"
+package.instdat.repo_name = "programs"
+package.instdat.repo_ref = "main"
+package.instdat.filemaps = {}
+package.instdat.filemaps["brainfuck.lua"] = "/bin/brainfuck.lua"
```
Comment (tomodachi94) [inline brainfuck.lua:11]: Looks like you missed something here.

**Review by tomodachi94 — CHANGES_REQUESTED** [review]

**Review by tomodachi94 — CHANGES_REQUESTED** [review]
This looks awesome! Just a few minor nitpicks before we merge.

**Review by tomodachi94 — CHANGES_REQUESTED** [review]
-- dupe --

**Review by tomodachi94 — COMMENTED** [review]
-- dupe --

File: brainfuck.lua [hunk 3]
```diff
@@ -0,0 +1,15 @@
+-- https://github.com/9551-Dev/programs
+
+local package = {}
+package.name = "graph"
```
Comment (tomodachi94) [inline brainfuck.lua]: ...and here.

File: basalt.lua (hunk 1 above)
Comment (Commandcracker) [reply basalt.lua:15]: this uses the minified version

File: brainfuck.lua (hunk 3 above)
Comment (Commandcracker) [reply brainfuck.lua]: Ohh, i mixed up files -_-

**Comment by tomodachi94** [comment]
Lgtm!

**Comment by tomodachi94** [comment]
Also @Commandcracker, quick question, what is "easyunicornpkg"?

Edit: Never mind, saw your message in MCCM.

**Pull request merged**